easybuild/easyblocks/j/java.py

```diff
@@ -27,13 +27,32 @@
 
 @author: Jens Timmerman (Ghent University)
 """
+from distutils.version import LooseVersion
 from easybuild.easyblocks.generic.packedbinary import PackedBinary
+from easybuild.tools.run import run_cmd
 
 
 class EB_Java(PackedBinary):
+#class PackedBinary(Binary, EasyBlock):
     """Support for installing Java as a packed binary file (.tar.gz)
     Use the PackedBinary easyblock and set some extra paths.
     """
+    
+    def extract_step(self):
+        """Unpack the source"""
+        #print LooseVersion(self.version)
+        #print self.src
+        print self.builddir
+        if LooseVersion(self.version) < LooseVersion('1.7'):
+            # uncompress java6 to build dir
+            cmd = 'cd %s; yes | bash %s' % (self.builddir, self.src[0]['path'])
```

Comment:
@pescobar alternatively, use the `inp` argument to `run_cmd` to specify input to `stdin`; you probably need to pass a space (rather than an empty string):

```python
run_cmd(cmd, log_all=True, simple=True, inp=' ')
```